### DISCUSSION TITLE
Topaz particle picking issue

### DISCUSSION THREAD

**Discussion by liningsdu** [tbepler/topaz#182] [cryoSPARC]
Hi,

Is there anyone have an idea why Topaz extract job only picked particles from left 1/4 area of my image? As shown in the picture, I can see all of the images were picked only left below corner. My cryosparc version is 4.3.1. At first I did the patched motion correction by F-crop 1/2, and I processed the data to almost reached the Nquist resolution, then I decided to redo motion correction job with no F-crop and CTF refinement. After that I tried to pick particles from new processed images, using the Topaz training model I trained on the binned data, this phenomenon happens. I also have tried re-train the Topaz model using particles from the un-bin data, but it still only picked left below corner area. Anything I’m wrong here? Thank you for your suggestions!

![image](https://github.com/tbepler/topaz/assets/35546299/7827b127-3dd3-463a-beb0-3c68474c618e)

**Comment by liningsdu** [comment]
BTW, I also tried the un-bin model which is imported from another project with similar structure shape. These two datasets are collected from same microscope with same settings. The trained model of another dataset was from the un-bin micrographs. But still only picked left below corner. Don’t know why this happens.

**Comment by tbepler** [comment]
This looks like a possible coordinate scaling problem. Are you plotting particle coordinates from a downsampled micrograph on the original full resolution micrograph?

**Reply by liningsdu** [reply]
Hi,
For both Topaz training and extract job, I set the downsampling factor to 4. The original pixel size is 1.1A.
